### DISCUSSION TITLE
modified phone number

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#522] [Q&A]
How do I indicate that person #X's phone number changed on a certain date? The same goes for having one phone number for home and another number for office.
I tried the notation below
1 ADDR
2 PHON 123-456-789
3 DATE FROM 1960 TO 1990
2 PHON 987-654-321
2 TYPE home
3 DATE FROM 1990 TO 2015
2 PHON 987-654-322
2 TYPE office
3 DATE FROM 1990 TO 2015
but this one was not accepted.

What are the alternatives?

Best regards,
Marianne van Harten

**Comment by tychonievich** [comment]
If the person is a contributing user (SUBM record) there is no standard support for storing old telephone numbers. You could use an extension tag for this if you wished, which I can explain more about upon request.

If the person is a historical individual (INDI record) then telephone numbers are associated with events or attributes, not with the INDI itself. If you wish to make a phone number its own attribute there is no single standard way to do that, but personally I think that PROP might be the best attribute for it (a phone property that belongs to a person) though I could see a case for using RESI or FACT instead. Using PROP, your example would become

```gedcom
0 @I1@ INDI
1 PROP telephone
2 PHON 123-456-789
2 DATE FROM 1960 TO 1990
1 PROP telephone
2 PHON 987-654-321
2 TYPE home
2 DATE FROM 1990 TO 2015
1 PROP telephone
2 PHON 987-654-322
2 TYPE office
2 DATE FROM 1990 TO 2015
```
